R/po_extract.R

```diff
@@ -0,0 +1,43 @@
+#' Extract messages for translation into a `.pot` file
+#'
+#' `po_extract()` scans your package for strings to be translated and
+#' saves them into a `.pot` template file (in the package's `po` directory).
+#' You should never modify this file by hand; instead modify the underlying
+#' source code and re-run `po_extract()`.
+#'
+#' @returns The extracted messages as computed by [get_message_data()],
+#'   invisibly.
+#' @inheritParams get_message_data
+po_extract <- function(
+    dir = ".",
+    custom_translation_functions = list(),
+    verbose = !is_testing(),
+    style = NULL) {
+
+  message_data <- get_message_data(dir,
+    custom_translation_functions = custom_translation_functions,
+    verbose = verbose,
+    style = style
+  )
+
+  n <- nrow(message_data)
+  if (!n) {
+    if (verbose) message('No messages to translate')
+    return(invisible())
+  }
+  if (verbose) messagef(ngettext(n, "Found %i message", "Found %i messages"), n)
+
+  po_dir <- file.path(dir, 'po')
+  dir.create(po_dir, showWarnings = FALSE)
+
+  desc <- get_desc_data(dir)
+  po_params = list(
+    package = desc[['Package']],
+    version = desc[['Version']],
+    copyright = NULL,
```

Comment:
is your sense that these fields (`copyright` and `bugs`) aren't really needed?

my sense as well has been these things are basically covered by the DESCRIPTION file already, and kept it to try & be consistent with `base` (and it's something of a maintenance headache to implement)

Reply:
Yeah, I think we should remove them; it's better to leave to the DESCRIPTION and avoid creating duplicates that might get out of date.